static/app/types/auth.tsx

```diff
@@ -91,11 +89,28 @@ export type Authenticator =
   | U2fAuthenticator
   | RecoveryAuthenticator;
 
+interface RegisterRequest {
+  appId: string;
+  challenge: string;
+  version: string;
+}
+
+interface SignRequest extends RegisterRequest {
```

Comment:
still surprised these aren't in DOM

Reply:
This is our own stuff coming back from the server